kimchi/wasm/src/plonk_verifier_index.rs

```diff
@@ -744,7 +744,7 @@ macro_rules! impl_verification_key {
                         },
                     };
 
-                let (linearization, powers_of_alpha) = expr_linearization(Some(&feature_flags), true);
+                let (linearization, powers_of_alpha) = expr_linearization(Some(&feature_flags), true, 3);
```

Comment:
What do you think about doing `let zk_rows = 3;` and using `zk_rows` everywhere there's a "magic" `3` occurring?